commons-profiler-api/src/main/java/ru/fix/commons/profiler/impl/ProfiledCallImpl.java

```diff
@@ -26,6 +26,13 @@
         this.profiledCallName = profiledCallName;
     }
 
+    @Override
+    public void call() {
+        profiler.applyToSharedCounters(profiledCallName, sharedCounters -> {
+            sharedCounters.getCallsCount().increment();
```

Comment:
You can also track max throughput here, it depends only on calls count and do not depend on latency.
Maybe it would be better to add it here? 
```
sharedCounters.getMaxThroughput().call();
```

Reply:
done!